apps/portalbase/system/system__purge/methodclass/system_purge.py

```
@@ -0,0 +1,39 @@
+from JumpScale import j
+from JumpScale.portal.portal.auth import auth
+
+class system_purge(j.code.classGetBase()):
+
+    """
+     purging of logs or jobs or ecos
+    
+    """
+
+    def __init__(self):
+
+        self._te = {}
+        self.actorname = "purge"
+        self.appname = "system"
+        self.scl = j.clients.osis.getNamespace('system', j.core.portal.active.osis)
+        pass
+
+
+    @auth(['level1', 'level2', 'level3'])
+    def purge(self, type, age, **kwargs):
+        try:
+             start = int(j.base.time.getEpochAgo(age))
+        except Exception:
+            return False
```

IMHO, you shouldn't hide the error, If the caller sent a wrong argument. He should get that in his face.